bluesky_queueserver/manager/manager.py

```diff
@@ -495,14 +498,23 @@ async def _start_plan_task(self, stop_queue=False):
 
         if not n_pending_plans:
             self._manager_state = MState.IDLE
+            self._worker_pause_pending = False
             success, err_msg = False, "Queue is empty."
             logger.info(err_msg)
 
         elif self._queue_stop_pending or stop_queue:
             self._manager_state = MState.IDLE
+            self._worker_pause_pending = False
             success, err_msg = False, "Queue is stopped."
             logger.info(err_msg)
 
+        elif self._worker_pause_pending:
+            self._worker_pause_pending = False
```

Comment:
I guess this line is redundant. It is repeated two lines below.

Reply:
woops! yes you're right, I will remove it